README.md

```diff
@@ -52,8 +52,16 @@ Please refer to the following instructions to setup Oppiabot for the first time
 6. [Create a new GitHub App](https://github.com/settings/apps/new) with:
     * **Webhook URL**: Use your `WEBHOOK_PROXY_URL` from the previous step.
     * **Webhook Secret**: `development`
-    * **Permissions & events** is located lower down the page and will depend on what data you want the bot to have access to. Currently, the bot requires read and write access to issues and pull requests and read-only access to the file in the path `.github/stale.yml`.
-    If, for example, you only enable issue events, you will not be able to listen on pull request webhooks with the bot.
+    * **Permissions & events** The following permissions and events must be subscribed.If, for example, you only enable issue events, you will not be able to listen on pull request webhooks with the bot.
+
+    <p align="center">
+    <img src="images/repo1-permissions.jpg" height="600" width="400">
```

Comment:
I think the images should be linked externally.
What do you think @vojtechjelinek.

Also, you should probably take new images as these are stretched since they are screenshots done on a mobile phone.

Reply:
I think it is fine if the images are in this repository, though maybe they should be added to something like docs/images/…, I agree that they should not be streched.

Reply:
I was thinking it would be better to add the images in the repository in docs/images folder and provide a link in the readme , because adding so many images is making the readme quite crowded.

Reply:
I think it's okay for the images to be stored in the repository, and it should also be shown in the readme so that contributors would be able to find the events they want quickly.